### PR TITLE
fix(hero-img-card): variable update

### PR DESCRIPTION
**Pull request by IdaHW**
Update colour variable for hero image card

# Description

Update Hero Image Card colour variable as was broken

# Checklist:

- [x] The commit messages follow the [convention for this project](./blob/master/docs/CONTRIBUTING.md#conventional-commits)
- [ ] I have provided an adequate amount of test coverage
- [ ] I have added the functionality to the [test app](./blob/master/docs/CONTRIBUTING.md#build-test-application)
- [ ] I have provided a story in storybook to document the changes
- [ ] I have provided documentation in the notes section of the story
- [ ] I have added any new public feature modules to public-api.ts

### CODE REVIEW

File: projects/canopy/src/lib/hero-img/hero-img-card/hero-img-card.component.scss [hunk 1]
```diff
@@ -1,7 +1,7 @@
 @import '../../../styles/mixins';
 
 .lg-hero-img-card {
-  background-color: var(--hero-img-bg-color-rgb);
+  background-color: var(--hero-img-bg-color);
```
Comment (elenagarrone) [inline projects/canopy/src/lib/hero-img/hero-img-card/hero-img-card.component.scss:4]: The rgb colour should work but i think you need to write:
```
background-color: rgba(var(--hero-img-bg-color-rgb), 1)
```

Comment (elenagarrone) [reply projects/canopy/src/lib/hero-img/hero-img-card/hero-img-card.component.scss:4]: BTW you can test this in your project by building the app in the dotcom `package.json` you can add this script:
```
"util:copy-canopy": "rimraf node_modules/@legal-and-general/canopy && copyfiles -u 3 \"../canopy/dist/canopy/**/*\" <your-folder-path>dotcom/node_modules/@legal-and-general/",
```

Comment (IdaHW) [reply projects/canopy/src/lib/hero-img/hero-img-card/hero-img-card.component.scss]: @elenagarrone thank you, that's a better solution :)

**Review by elenagarrone — APPROVED** [review]

File: projects/canopy/src/lib/hero-img/hero-img-card/hero-img-card.component.scss (hunk 1 above)
Comment (IdaHW) [reply projects/canopy/src/lib/hero-img/hero-img-card/hero-img-card.component.scss]: Thanks @elenagarrone - we've also implemented a script that copied the local version of canopy to the dotcom project. But perhaps your solution is easier to work with

**Review by owensgit — APPROVED** [review]

**Review by elenagarrone — APPROVED** [review]

**Pull request merged**